### PR TITLE
Docs: reintroduce `aria-describedby` advice for help text

### PR DESCRIPTION
**Pull request by patrickhlauke**
Based on v3 http://getbootstrap.com/css/#forms-help-text, adapted to make examples more "real life"

### CODE REVIEW

**Comment by patrickhlauke** [comment]
capture

File: docs/components/forms.md [hunk 1]
```diff
@@ -517,19 +517,31 @@ Wrap inputs in grid columns, or any custom parent element, to easily enforce des
...
+</form>
 {% endexample %}
 
 Block help text—for below inputs or for longer lines of help text—can be easily achieved with a `<p>`.
 
 {% example html %}
-<p class="text-muted">
-  A block of help text that breaks onto a new line and may extend beyond one line.
+<label for="inputPassword2">Password</label>
+<input type="password" id="inputPassword2" class="form-control" aria-describedby="passwordHelpBlock">
```
Comment (cvrebert) [inline docs/components/forms.md]:
```
Running "htmllint:src" (htmllint) task
_gh_pages/components/forms/index.html [L1679:C103] Duplicate ID “inputPassword2”.
```

I'd recommend using `inputPassword4` and `inputPassword5` as the IDs in these new examples instead.

Comment (patrickhlauke) [inline docs/components/forms.md]: ah, good catch (related, I really need to sort out my eslint which keeps giving me weird errors in unrelated files - js/src/scrollspy.js - as that's what's been keeping me from running local tests)

**Comment by cvrebert** [comment]
Seems like these should use `.form-group` for consistency with the other form examples.

**Comment by patrickhlauke** [comment]
Added a `<div class="form-group">` to the first example, as that also had the full `<form>` around it. For the second one, I'd say it can remain without the `<div>`, as it's only a fragment and not a complete form (similar to some of the more focused previous examples like radio buttons etc)

Also spotted two instances of help text in the main form controls example which lacked `aria-describedby`...rectified now
